doc/api/esm.md

```diff
@@ -1551,10 +1588,11 @@ The resolver can throw the following errors:
   or package subpath specifier.
 * _Invalid Package Configuration_: package.json configuration is invalid or
   contains an invalid configuration.
-* _Invalid Package Target_: Package exports define a target module within the
-  package that is an invalid type or string target.
+* _Invalid Package Target_: Package exports or imports define a target module
```

Comment:
"target module *within the package*" may be outdated with `imports`?

Reply:
Good point, changed.